app/views/shared/smart_annotation/_menu.html.erb

```diff
@@ -18,7 +18,9 @@
           <% if repository.shared_with?(current_team) %>
             <i class="sn-icon sn-icon-user-menu-friends"></i>
           <% end %>
-          <%= repository.name %>
+          <div class="max-w-[150px] overflow-hidden whitespace-nowrap ml-[-10px] text-left truncate">
```

Comment:
truncate already include nowrap and overflow hidden.
And check tailwind documentation this margin already exists: -ml-2.5